### ISSUE TITLE
depth_to_class function is missing & loss function issue

### ISSUE THREAD

**Issue by SerinaWei** [YvanYin/VNL_Monocular_Depth_Prediction#36]
Thanks YvanYin for the very nice work! I was trying to train kitti data with the provided loss function, but notice that in the kitti_dataset.py, the depth_to_class() module is missing. Therefore we cannot get data['b_classes']. In addition, I wonder if in the loss function, the input to the cross entropy loss should be (pred_depth, data['B_classes'], etc)?

Thanks in advance for the help!

**Comment by YvanYin** [comment]
1. You can directly use the depth_to_class() from nyu_dataset.py. 
2. The input to cross entropy loss likes this: loss_metric = self.weight_cross_entropy_loss(pred_logit, data['B_bins'], data['B'].cuda())

**Comment by SerinaWei** [comment]
Thanks for the clarification!

**Comment by andyhahaha** [comment]
Hi YvanYin,
Thanks for your great work!
I still can't find the depth_to_class() function in nyu_dataset.py.
Where can I found it. Thanks!
